status.py

```diff
@@ -9,8 +9,15 @@
 def run(event: Dict[str, Any], context: 'bootstrap.LambdaContext') -> Dict[str, Any]:
     """ Get mined tokens from the full node and return the economics status of the network
     """
+    # Default for status with no decimals parameter is true
+    decimals = True
```

Comment:
Shouldn't we allow a `comma` parameter as well? It may be true by default, but we will be able to choose what format we'd like to receive.

Reply:
I'm still not sure about it, I agree with @jansegre (https://github.com/HathorNetwork/hathor-economics-api/pull/6#pullrequestreview-444862603) that we should not add many format options on the server, this should be the clients job. I think we've done this decimals+thousands separator just because we have no client to show this info and it's easier for people to see the economics status this way.

Reply:
But it makes things less computer friendly. People would have to parse it instead of getting it directly parsed by the JSON reader. If we don't add a parameter, I think we shouldn't add comma separators at all.

Reply:
@msbrogli I don't understand, we have the API with the raw value already, I thought it was meant for that. Why does the decimal option API exist then? If it's just for a /100 division, we could return a Number instead of a String.

Technically it's also possible to use Python's Decimal type and implement a JSON serializer (not a full serializer just a function passed to `json.dumps` that special cases `decimal.Decimal`) to render it as a Number with the exact decimal places we want (that is, always `.00`). But this may be an overkill depending on who should be the user of this API.

Reply:
Yeah, I kind of agree with @jansegre. We need to decide what's the goal of adding the decimals, if it's to make easier to read the number, I think the thousand separator is also valid. We already have an API if people want to parse it.

Maybe we should not add this decimals and thousand separator here and add the data on the explorer, so in the frontend we parse the number and format it.

Reply:
Let's keep it simple. Our immediate need is the economics API and people will use it to view the information directly, so it's better to use commas. If someone has the need for this information without it, we can add later, as well as adding this to the explorer.

Reply:
It's also pretty simple to get rid of the commas, `.replace(',', '')`. Which is what you could do if you want to parse the values when using `decimals=true`.

Reply:
Fair point! Anyone can use the API without decimals. I'm good with the way it is developed.